samples/cellular/uicc_lwm2m/sample.yaml

```
@@ -0,0 +1,18 @@
+sample:
+  name: UICC LwM2M sample
+tests:
+  samples.cellular.uicc_lwm2m:
```

Name of the sample scenario should start with `sample`:
```suggestion
  sample.cellular.uicc_lwm2m:
```